### DISCUSSION TITLE
TMA value for temperatures

### DISCUSSION THREAD

**Discussion by oh2th** [goecharger/go-eCharger-API-v2#93] [Q&A]
I am wondering could someone update the API Keys document to describe what are the two temperature values in the array for `tma`?

**Comment by ICarryTheDustOfAJourney** [comment]
By comparing to the values provided to the values shown in the apps's graph, one could guess that 

tma[0] = "Typ 2" (= temperature of the cable? one of the 2 connectors? )
tma[1] = "Stomversorgung"  (=power supply, probably somewhere in the box)

May be someone with insights can shed some light on this?

**Reply by 0xFEEDC0DE64** [reply]
@peterpoetzi stimmt das?
